geoutils/georaster/raster.py

```diff
@@ -491,7 +563,7 @@ def __add__(self: RasterType, other: RasterType | np.ndarray | Number) -> Raster
         out_data = self_data + other_data
 
         # Check that if no ndv was set, a default value is used
-        if (np.sum(out_data.mask) > 0) & (ndv is None):
+        if (np.sum(getattr(out_data, "mask", 0)) > 0) & (ndv is None):
```

Comment:
why was this change needed?

Reply:
Because it's not necessarily a masked array. Related to your suggestion if it should maybe always be one.

Reply:
Reverted this change in 3f95a7e since it's no longer needed.